### PR TITLE
Faster eo3 conversion

### PR DESCRIPTION
**Pull request by Ariana-B**
Make remove converter initialisation out of `from_doc` to make it faster, and remove `normalise_properties` param as it is not being used.

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -71,6 +71,7 @@
         "ciso8601",
         "click",
         "defusedxml",
+        "h5py",
```
Comment (omad) [inline setup.py:79]: This is already covered up able as an extra dependency for "all" and "wagl", so shouldn't be necessary here.

**Review by omad — CHANGES_REQUESTED** [review]
Looks good!

I search a bit further and couldn't find anything using the `normalise_properties` argument in other repos either. However, we need some sort of a changelog, to record for the next release that the API has changed.

Can you please do a rebase/force-push to remove these two extra files in `dist/` and we can merge.

File: setup.py (hunk 1 above)
Comment (Ariana-B) [reply setup.py:79]: This was a change I previously made as part of the conda-forge deployment; without it, their pipeline failed

Comment (omad) [reply setup.py:79]: Okay, thanks, that sounds fair enough.

Is an indicator that we need to be careful about what/when tests are run more than anything I think.

**Review by omad — APPROVED** [review]
TY!

**Pull request merged**